doc/persisted-queries.md

```diff
@@ -0,0 +1,17 @@
+# Persisted queries
+
+## Automatic persisted queries
+
+Apollo supports [Automatic Persisted Queries](https://www.apollographql.com/docs/apollo-server/performance/apq/).
+
+// TODO: add some sample code
```

Comment:
Is there any sample code really? I think it may be just enabling the flag in Gradle. The rest will be handled.

Reply:
My bad, it's a bad merge, there's already more documentation under `docs/source/advanced/persisted-queries.md`. I'll remove that file.